src/source/reporting.rst

```diff
@@ -68,46 +68,49 @@ The dimensions supported by the reporting API have discrete values (like *Nether
-``am:sessionECPC``                 Long       Session    Effective cost per unique sessions with website click. Total spent on the ad divided by the number of unique sessions with website clicks
-``am:sessionsWithClicks``          Long       Session    Number of unique sessions with clicks
-``am:sessionsWithImpressions``     Long       Session    Number of unique sessions with impressions
-``am:sessionsWithWebsiteClicks``   Long       Session    Number of unique sessions with website clicks
-``am:sessionsWithEmails``          Long       Session    Number of unique sessions sending emails
-``am:sessionsWithEngagements``     Long       Session    Number of unique sessions with engagements
-``am:sessionViewCTR``              Double     Session    Number of unique sessions with clicks leads from unique sessions with impressions
-``am:sessionWebsiteCTR``           Double     Session    Number of unique sessions with website leads from unique sessions with clicks
-``am:sessionEngagementCTR``        Double     Session    Number of unique sessions with engagements leads from unique sessions with clicks
-================================  =========  =========  ============================================================
+================================  =========  =========  =========  ======================================
+Name                              Type       Version    `Scope`_   Description
+================================  =========  =========  =========  ======================================
+``am:clicks``                     Long       V1,V2      Hit        Number of clicks
+``am:impressions``                Long       V1,V2      Hit        Nnumber of impressions
+``am:websiteClicks``              Long       V1,V2      Hit        Number of website clicks
+``am:emails``                     Long       V1,V2      Hit        Number of emails
+``am:engagements``                Long       V1,V2      Hit        Number of engagements from clicks
+``am:viewCTR``                    Double     V1,V2      Hit        Number of clicks leads from impressions, also known as Click-through rate
+``am:websiteCTR``                 Double     V1,V2      Hit        Number of website leads from clicks
+``am:spent``                      Long       V1         Hit        Amount spent (in Cents)
+``am:spentMicros``                Long       V2         Hit        Amount spent (in Micros)
+``am:engagementCTR``              Double     V1,V2      Hit        Number of engagements leads from clicks
+``am:avgCPC``                     Double     V1         Hit        Average Cost Per Click. Calculation
+``am:avgBidMicros``               Double     V2         Hit        Average Cost Per Click in micros. Calculation
+``am:avgBilledMicros``            Double     V2         Hit        Average Billed Cost Per Click in micros. Calculation
```

Comment:
unfinished `Calculation` sentence. Something looks to be broken since before your PR. I remember the calculation being there but not sure at which point it got broken.